js/bitvavo.js

```diff
@@ -133,38 +136,6 @@ module.exports = class bitvavo extends Exchange {
                     },
                 },
             },
-            'fees': {
```

Comment:
Let's restore these fees and just fix them for their current numbers from here: https://bitvavo.com/en/fees